internal/pkg/plugin/githubactions/golang/templates.go

```diff
@@ -23,14 +23,14 @@ jobs:
       with:
         go-version: 1.17
     - name: Test
-      run: [[- if not .Test.Command]] go test ./...[[- else]][[.Test.Command]][[- end]] [[- if .Test.Coverage.Enable]] -race -covermode=atomic -coverprofile=[[- if not .Test.Coverage.Output]]coverage.out[[- else]][[.Test.Coverage.Output]][[- end]] [[- end]]
+      run: [[if not .Test.Command]] go test ./...[[- else]][[.Test.Command]][[- end]] [[- if .Test.Coverage.Enable]] -race -covermode=atomic -coverprofile=[[- if not .Test.Coverage.Output]]coverage.out[[- else]][[.Test.Coverage.Output]][[- end]] [[- end]]
```

Comment:
[[if not .Test.Command]]
the format is different from
[[ if not .Build.Command]]

space or not? one space at the beginning only? or one space at the beginning, one at the end? 

Make it consistent.

Also, please test both with coverate.output and without, and paste the screenshot here.